Specs/Scene/ImageryLayerCollectionSpec.js

```diff
@@ -258,6 +258,260 @@ describe(
       expect(collection.isDestroyed()).toEqual(true);
     });
 
+    describe("pickImageryLayers", function () {
+      var scene;
+      var globe;
+      var camera;
+
+      beforeAll(function () {
+        scene = createScene();
+        globe = scene.globe = new Globe();
+        camera = scene.camera;
+
+        scene.frameState.passes.render = true;
+      });
+
+      afterAll(function () {
+        scene.destroyForSpecs();
+      });
+
+      beforeEach(function () {
+        globe.imageryLayers.removeAll();
+      });
+
+      /**
+       * Repeatedly calls update until the load queue is empty.  Returns a promise that resolves
+       * once the load queue is empty.
+       */
+      function updateUntilDone(globe) {
+        // update until the load queue is empty.
+        return pollToPromise(function () {
+          globe._surface._debug.enableDebugOutput = true;
+          scene.render();
+          return (
+            globe._surface.tileProvider.ready &&
+            globe._surface._tileLoadQueueHigh.length === 0 &&
+            globe._surface._tileLoadQueueMedium.length === 0 &&
+            globe._surface._tileLoadQueueLow.length === 0 &&
+            globe._surface._debug.tilesWaitingForChildren === 0
+          );
+        });
+      }
```

Comment:
Is it possible to move this helper function outside of the `describe` block so that it can be used as a helper both here and  in `pickImageryLayerFeatures`? `scene` will need to be passed in as a parameter is all.